integration_tests/fixtures/forseti/main.tf

```diff
@@ -41,11 +41,12 @@ module "bastion" {
 }
 
 module "forseti" {
-  source = "github.com/forseti-security/terraform-google-forseti"
```

Comment:
@kevensen Does this change look ok? By using the github link does that mean we are using the master branch of the module? If so, then maybe I should revert this part and continue using master branch of the tf module.

Reply:
@gkowalski-google From what I understand about the [GitHub Terraform source](https://www.terraform.io/docs/modules/sources.html#github), yes it would pull from master.

Reply:
I have reverted this change, will use master tf module.